tutorials/advanced_features/State Preparation with the SLM Mask.ipynb

```diff
@@ -19,7 +19,8 @@
    "metadata": {},
```

Comment:
<p>I changed this too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>